reVX/least_cost_xmission/cost_creator.py

```diff
@@ -239,7 +245,7 @@ def compute_multipliers(self, iso_mults, excl_h5=None,
 
         # Set water multiplier last so we don't get super high multipliers at
         # water body boundaries next to steep slopes
-        mults_arr[land_use == WATER_NLCD_CODE] = WATER_MULT
+        mults_arr[land_use == 11] = 10
```

Comment:
The 10 should probably be a kwarg. Using WATER_NLCD_CODE is a bit more obvious than 11, though the meaning can be derived pretty quickly from the comment. I'm indifferent.